<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="main" stretch="1,0,10,1">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="top" stretch="1,5,1">
       <item>
        <widget class="QPushButton" name="back">
         <property name="text">
          <string>BACK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="title">
         <property name="text">
          <string>Attendance Tracking</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="middle" stretch="1,7">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="filter_container">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <layout class="QFormLayout" name="Section_layout">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
           </property>
           <property name="rowWrapPolicy">
            <enum>QFormLayout::WrapAllRows</enum>
           </property>
           <property name="labelAlignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="formAlignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="course_label">
             <property name="text">
              <string>Section</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="section_selection">
             <property name="placeholderText">
              <string>Select a section</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QFormLayout" name="group_layout">
           <property name="rowWrapPolicy">
            <enum>QFormLayout::WrapAllRows</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="group_label">
             <property name="text">
              <string>Group</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="group_selection">
             <property name="placeholderText">
              <string>Select a group</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QFormLayout" name="activity_layout">
           <property name="rowWrapPolicy">
            <enum>QFormLayout::WrapAllRows</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="activity_label">
             <property name="text">
              <string>Activity</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="activity_selection">
             <property name="placeholderText">
              <string>Select an Activity</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QFormLayout" name="date_layout">
           <property name="rowWrapPolicy">
            <enum>QFormLayout::WrapAllRows</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="date_label">
             <property name="text">
              <string>Date</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QDateEdit" name="date_selection">
             <property name="dateTime">
              <datetime>
               <hour>0</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2026</year>
               <month>1</month>
               <day>1</day>
              </datetime>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="table_container">
         <item>
          <widget class="QTableWidget" name="table">
           <property name="verticalScrollMode">
            <enum>QAbstractItemView::ScrollPerPixel</enum>
           </property>
           <property name="horizontalScrollMode">
            <enum>QAbstractItemView::ScrollPerPixel</enum>
           </property>
           <property name="rowCount">
            <number>0</number>
           </property>
           <property name="columnCount">
            <number>0</number>
           </property>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="bottom" stretch="5,1,1">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="mark_all">
         <property name="text">
          <string>Mark All Present</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="save">
         <property name="text">
          <string>Save Attendance</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
